INSTALL.rst

```diff
@@ -257,6 +257,27 @@ location specified in ``haas.cfg``) and initialize its tables::
 
   haas init_db
 
+
+Authentication and Authorization
+--------------------------------
+
+HaaS includes a pluggible architecture for authentication and authorization.
+Right now, the only authentication backend uses HTTP basic auth, with usernames
+and password stored in the database. to enable this, Make sure the
```

Comment:
Fixed the capitalization. I clarified this section, and documented the presence of the null auth backend, but I actually think it's a really horrible idea to have "no auth" be the default. I added a check (which should have been there before, but I missed it) to verify that *some* auth backend was loaded.

Also, due to the fact that this is something that *must* be configured, I think putting it in an extensions subsection would be too obscure.